piwik-sdk/src/main/java/org/piwik/sdk/ecommerce/EcommerceItems.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Android SDK for Piwik
+ *
+ * @link https://github.com/piwik/piwik-android-sdk
+ * @license https://github.com/piwik/piwik-sdk-android/blob/master/LICENSE BSD-3 Clause
+ */
+
+package org.piwik.sdk.ecommerce;
+
+import org.piwik.sdk.tools.CurrencyFormatter;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class EcommerceItems {
+    public Map<String, List<String>> items = new HashMap<>();
+
+    public void addItem(String sku, String name, String category, int price, int quantity) {
+        List<String> item = Arrays.asList(sku, name, category, CurrencyFormatter.priceString(price), "" + quantity);
+        items.put(sku, item);
+    }
+
+    public String toJson() {
```

Comment:
Better use JSON classes for this, see:
https://github.com/piwik/piwik-sdk-android/blob/dev/piwik-sdk/src/main/java/org/piwik/sdk/CustomVariables.java